generators/traefik/traefik.go

```diff
@@ -143,43 +143,43 @@ func (g *Generator) Generate(opts *options.Options, spec *openapi3.T) (string, e
 	// Main routine
 	// Iterate on all paths and build routes rules with related middlewares and any overrides
 	for path, pathItem := range spec.Paths {
-		// x-kusk options per path
-		// ServersTransport for this path
-		pathServiceServersTransport := serviceServersTransport
-		// Create copy of root middlewares map to further override per path
-		pathMiddlewares := copyMiddlewareMap(rootMiddlewares)
-		// x-kusk options per path
-		pathSubOpts, ok := opts.PathSubOptions[path]
-		if ok {
-			if opts.IsPathDisabled(path) {
+		// x-kusk options per operation (http method)
+		// For each method we create separate Match rule and route and then add to routes list
+		for method := range pathItem.Operations() {
```

Comment:
This is not optimal, moving "for" here runs parsing for pathOpts each time for each method in the path, while only needs to be done once per path. Since we're using "reflect" for this this will hurt performance. We will be also generating e.g. path CORS middleware every time and copying root map every time.

Reply:
Ah, yes you're right!

My thought process was that we should only be generating it if we're actually using it i.e. the operation is not disabled. But it's better to generate it once per path for sure. I will put this back then 👍